checker/tests/nullness/ArrayMainModifierNullableAnnotation.java

```diff
@@ -0,0 +1,8 @@
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class ArrayMainModifierNullableAnnotation {
```

Comment:
I would probably put the two test cases into one file and name it `ArrayCreation` or something like that.